projects/ltc2378/README.rst

```diff
@@ -0,0 +1,162 @@
+input ranges. The board provides all necessary hardware connections and signal
+conditioning for comprehensive ADC evaluation.
+
+The evaluation board provides easy access to all necessary signals through
+standard connectors, enabling quick setup for high-speed data acquisition
+applications up to 1Msps sample rate.
+
+For full performance details, refer to the LTC2378-20 data sheet, which should
+be consulted in conjunction with the user guide.
+
+Hardware Specifications
+-----------------------
+
+Power Supply Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The DC2135A evaluation board requires external ±16V supplies for the analog
+input circuitry and signal conditioning. The LTC2378-20 ADC itself operates
+from a regulated 2.5V supply that is generated on-board.
+
+**Pin Description**
+
+	Please see the following table for the pin assignments:
+
+	+----------+-------------------------------------------+
+	| Name     | Description			       |
+	+----------+-------------------------------------------+
+	| GND      | Connect to Ground			       |
+	+----------+-------------------------------------------+
+	| SCK      | Connect to SPI Clock (SCK)		       |
+	+----------+-------------------------------------------+
+	| SDO      | Connect to SPI Master In Slave Out (MISO) |
+	+----------+-------------------------------------------+
+	| SDI      | Connect to SPI Master Out Slave In (MOSI) |
+	+----------+-------------------------------------------+
+	| CNV      | Connect to GPIO pin (CNV)		       |
+	+----------+-------------------------------------------+
+	| BUSY     | Connect to GPIO pin (BUSY)		       |
+	+----------+-------------------------------------------+
+
+**Hardware Bringup**
+
+For reference, consult the Quick Start Procedure section in the user guide for the corresponding demo board:
+`DC2135A user guide <https://www.analog.com/media/en/technical-documentation/user-guides/DC2135AF.PDF>`_.
+
+No-OS Build Setup
+-----------------
+
+Please see: https://wiki.analog.com/resources/no-os/build
+
+No-OS Supported Examples
+------------------------
+
+The initialization data used in the examples is taken out from:
+`Project Common Data Path <https://github.com/analogdevicesinc/no-OS/tree/main/projects/ltc2378/src/common>`_
+
+The macros used in Common Data are defined in platform specific files found in:
+`Project Platform Configuration Path <https://github.com/analogdevicesinc/no-OS/tree/main/projects/ltc2378/src/platform>`_
+
+Basic example
+^^^^^^^^^^^^^
+
+This is a simple example that initializes the LTC2378-20, perform the start-up
+sequence and configure the output voltage.
+
+In order to build the basic example make sure you have the following
+configuration in the
+`Makefile <https://github.com/analogdevicesinc/no-OS/tree/main/projects/ltc2378/Makefile>`_
+
+.. code-block:: bash
+
+	# Select the example you want by choosing basic or iio_example
+	EXAMPLE ?= basic
```

Comment:
user should select either with 
```
make EXAMPLE=iio
```

or

```
export EXAMPLE=iio
make
```

whichever you want, but they shouldn't edit the Makefile